apps/catalogue/package.json

```diff
@@ -8,11 +8,23 @@
     "lint": "vue-cli-service lint"
   },
   "dependencies": {
+    "@fortawesome/fontawesome-svg-core": "^1.2.35",
+    "@fortawesome/free-solid-svg-icons": "^5.15.3",
+    "@fortawesome/vue-fontawesome": "^2.0.2",
+    "@molgenis-ui/components-library": "^2.4.5",
     "@mswertz/emx2-styleguide": "*",
     "core-js": "3.11.2",
+    "bootstrap-vue": "^2.21.2",
```

Comment:
i thought @mswertz idea was to use a custom build comp lib ? , i'm not against using bs-vue but is does lock us into the bs-vue - release cycle.

Reply:
I think it is a good idea to lean on the filters we already made and pull them into this repository. It is a nice framework which can help us a lot. The filters we created for dataexplorer 2 require bootstrap vue